packages/rum-core/src/common/constants.js

```diff
@@ -60,6 +60,10 @@ const RESOURCE_INITIATOR_TYPES = [
  * Maximum duration of the span that is used to decide if the span is valid - 300 secs / 5 mins
  */
 const SPAN_THRESHOLD = 5 * 60 * 1000
+/**
+ * Check only for long tasks that are more than 60ms
+ */
+const USER_TIMING_THRESHOLD = 60
```

Comment:
Can there be a lot of small tasks that eventually slows down the app? we have a method `groupSmallContinuouslySimilarSpans` in PerformanceMonitoring that groups small spans into one span if they repeat continuously!

Reply:
Well these are profiled code that user is doing in the application at different times while the application is loading and name of the spans are different for each perfEntry which ideally skips the similar span check.

Reply:
Why the span names would be different each time?

Reply:
`performance.measure("foo"), performance.measure("bar")`, this is how the entries will be added by the user. We are not capturing marks, only the measures.